tests/keras_tests/non_parallel_tests/test_keras_tp_model.py

```diff
@@ -252,20 +252,16 @@ def test_get_keras_tpc(self):
         def rep_data():
             yield [np.random.randn(*input_shape)]
 
-        quantized_model, _ = mct.ptq.keras_post_training_quantization_experimental(model,
-                                                                                   rep_data,
-                                                                                   target_platform_capabilities=tpc,
-                                                                                   new_experimental_exporter=True)
-
-        core_config = mct.core.CoreConfig(
-            mixed_precision_config=mct.core.MixedPrecisionQuantizationConfig(num_of_images=1,
-                                                                             use_hessian_based_scores=False))
-        quantized_model, _ = mct.ptq.keras_post_training_quantization_experimental(model,
-                                                                                   rep_data,
-                                                                                   core_config=core_config,
-                                                                                   target_kpi=mct.core.KPI(np.inf),
-                                                                                   target_platform_capabilities=tpc,
-                                                                                   new_experimental_exporter=True)
+        quantized_model, _ = mct.ptq.keras_post_training_quantization(model,
+                                                                      rep_data,
+                                                                      target_platform_capabilities=tpc)
+        core_config = mct.core.CoreConfig(mixed_precision_config=mct.core.MixedPrecisionQuantizationConfig(num_of_images=1,
```

Comment:
same for number of images (I know that it's not part of the API change, but if you change this and it doesn't break anything that requires debug maybe use this chance and update the test)